### ISSUE TITLE
Pillow version 9 error

### ISSUE THREAD

**Issue by FatihSarigol** [higlass/higlass-server#159]
Hi! 
My docker-build command:
`docker build -t higlass-server -f docker-context/Dockerfile .
`
ran into the error below:

```
Collecting pandas>=0.23.4
  Downloading pandas-1.1.5-cp36-cp36m-manylinux1_x86_64.whl (9.5 MB)
ERROR: Could not find a version that satisfies the requirement Pillow>=9.0.0 (from versions: 1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.7.4, 1.7.5, 1.7.6, 1.7.7, 1.7.8, 2.0.0, 2.1.0, 2.2.0, 2.2.1, 2.2.2, 2.3.0, 2.3.1, 2.3.2, 2.4.0, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.6.0, 2.6.1, 2.6.2, 2.7.0, 2.8.0, 2.8.1, 2.8.2, 2.9.0, 3.0.0, 3.1.0rc1, 3.1.0, 3.1.1, 3.1.2, 3.2.0, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.4.0, 3.4.1, 3.4.2, 4.0.0, 4.1.0, 4.1.1, 4.2.0, 4.2.1, 4.3.0, 5.0.0, 5.1.0, 5.2.0, 5.3.0, 5.4.0, 5.4.1, 6.0.0, 6.1.0, 6.2.0, 6.2.1, 6.2.2, 7.0.0, 7.1.0, 7.1.1, 7.1.2, 7.2.0, 8.0.0, 8.0.1, 8.1.0, 8.1.1, 8.1.2, 8.2.0, 8.3.0, 8.3.1, 8.3.2, 8.4.0)
ERROR: No matching distribution found for Pillow>=9.0.0
The command '/bin/sh -c pip install -r requirements.txt' returned a non-zero code: 1

```

When I try to install it myself, I receive the same error:

`python3 -m pip install Pillow==9.0.0`

```
ERROR: Could not find a version that satisfies the requirement Pillow==9.0.0 (from versions: 1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7.0, 1.7.1, 1.7.2, 1.7.3, 1.7.4, 1.7.5, 1.7.6, 1.7.7, 1.7.8, 2.0.0, 2.1.0, 2.2.0, 2.2.1, 2.2.2, 2.3.0, 2.3.1, 2.3.2, 2.4.0, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.6.0, 2.6.1, 2.6.2, 2.7.0, 2.8.0, 2.8.1, 2.8.2, 2.9.0, 3.0.0, 3.1.0rc1, 3.1.0, 3.1.1, 3.1.2, 3.2.0, 3.3.0, 3.3.1, 3.3.2, 3.3.3, 3.4.0, 3.4.1, 3.4.2, 4.0.0, 4.1.0, 4.1.1, 4.2.0, 4.2.1, 4.3.0, 5.0.0, 5.1.0, 5.2.0, 5.3.0, 5.4.0, 5.4.1, 6.0.0, 6.1.0, 6.2.0, 6.2.1, 6.2.2, 7.0.0, 7.1.0, 7.1.1, 7.1.2, 7.2.0, 8.0.0, 8.0.1, 8.1.0, 8.1.1, 8.1.2, 8.2.0, 8.3.0, 8.3.1, 8.3.2, 8.4.0)
ERROR: No matching distribution found for Pillow==9.0.0

```

The reason of this is as I found out a recent change:

> Pillow has dropped support for Python 3.6, which reached end-of-life on 2021-12-23.
https://pillow.readthedocs.io/en/stable/releasenotes/9.0.0.html

And my python version is also 3.6. This happens to increase higlass python version dependency from 3.6 to 3.7 I suppose, I will try to install it on another version and see if Higlass will work.

Thanks

https://github.com/higlass/higlass-server/pull/153

**Comment by pkerpedjiev** [comment]
Huh, it did install in the official higlass Docker container:

https://github.com/higlass/higlass-docker/runs/5243233636?check_suite_focus=true#step:3:3005

But that uses Python 3.8. Did you end up resolving your issue?

**Comment by FatihSarigol** [comment]
Thank you Peter!
Yes it was happening because of my python version. I suppose it should not work anywhere anymore with 3.6 because Pillow doesn't support that anymore. I tried my luck with changing the versions in the docker file myself but that led to a chain of other dependency version issues, and even when I installed a python 3.9 version to the server it again failed somehow because of some other thing that I honestly don't remember now, but in the meantime I thought about trying https://github.com/higlass/higlass-docker which did work so I used that on our server.
Thanks!
